.circleci/config.yml

```diff
@@ -12,6 +12,17 @@ jobs:
       environment:
         xpack.security.enabled: false
         transport.host: localhost
+    - image: rabbitmq:3.8.19
```

Comment:
do you still need the rabbitmq configuration?

Reply:
No, I'll remove this.